packages/@sanity/default-layout/src/components/LoginStatus.js

```diff
@@ -64,6 +64,7 @@ class LoginStatus extends React.PureComponent {
                 src={user.profileImage}
                 className={styles.userImage}
                 alt={`${user.name}'s profile image`}
+                data-initials={user.name ? user.name.charAt(0) : user.email.charAt(0)}
```

Comment:
Is it possible that `user.email` is missing here?

To be extra safe and future proof this can be written
`(user.name || user.email || '?').charAt(0)`